### PR TITLE
project-movie by Birgit Nehrwein and Jenny Quach

### PR DESCRIPTION
**Pull request by jenquach**
Link to my deployed project: https://serene-ride-8ae74a.netlify.app

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+            <Route exact path="/">
+              <MovieList 
+              />
+            </Route>  
+            <Route exact path="/upcoming">
+              <MovieUpcoming 
+              />
+            </Route>
+            <Route path="/movies/:movieId" component={MovieDetails} />
+            <Route path="/404" component={NotFound} />
```
Comment (DALA746) [inline code/src/App.js:25]: different approach for routes, next time try to be consistent with your code and use only one of approaches, consistency is a key :)

Comment (DALA746) [inline code/src/App.js:25]: different approach for routes, next time try to be consistent with your code and use only one of approaches, consistency is a key :)

Comment (DALA746) [inline code/src/App.js:25]: different approach for routes, next time try to be consistent with your code and use only one of approaches, consistency is a key :)

Comment (DALA746) [inline code/src/App.js:25]: different approach for routes, next time try to be consistent with your code and use only one of approaches, consistency is a key :)

File: code/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  text-align: center;
+  font-size: 18px;
+  margin-top: auto;
+  padding-top: 2px;
+  font-weight: 600;
+`
+
+const Footer = () => <StyledFooter>© BumBumBurka & JenQuach</StyledFooter>
+
+export default Footer
```
Comment (DALA746) [inline code/src/components/Footer.js:16]: it looks pretty good to have styled component, clean and organized code. It's nice to keep everything together both styling and logic at the same place. Looking forward to learn more about styled components now :)

File: code/src/components/Movie.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/* import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
```
Comment (DALA746) [inline code/src/components/Movie.js:3]: This file is commented out but exists in the repository. Why?

**Review by DALA746 — COMMENTED** [review]
Really nice job on this project! Your project looks good and fulfills the requirements :) I really like how your code looks, it is easy to read, it is clean and structured in a good way. By using styled components makes code looks even cleaner, it is easy to find the correct styling in css instead of scrolling for some time in main css file. Nice job! I am looking forward to see more of your future projects :)

**Review by MariaThomasson — COMMENTED** [review]
Really nice project Jenny & Birgit. The code is nicely structure and everything works fine. I enjoyed to see styled components. I would love to see some comments, especially in the larger files.
Darya already gave you some comments for the code so I focused on looking at the commits and the inspector tool. 

You are really good at explaining the updates you do and the commits messages and I think you both should keep up the good work with that and continue with improvement though it will be very helpful for future projects. (And to go back to your own project and understand what you did :)

I also tried out your Netlify-link with the inspector tool and Lighthouse. I did not have any comments on my own for your code but with Lighthouse a few suggestions appear if you want to improve your code even more and also make it more accessible. 

Skärmavbild 2021-11-10 kl  12 30 02
Skärmavbild 2021-11-10 kl  12 30 48
Skärmavbild 2021-11-10 kl  12 29 33
 If you have any questions please let me know.
 /Maria

**Pull request closed without merging**